### ISSUE TITLE
mscx2music converter throwing error

### ISSUE THREAD

**Issue by UnhappyCodings** [Ocawesome101/minecraft-pipe-organ#4]
Hi!

First of all nice work done here!
Sadly I get this error when I even try to execute the mscx2music command:
![image](https://github.com/user-attachments/assets/de03fa95-1765-4f27-9dd2-0b8eae7a6560)

Is this known and maybe even fixed, saw you commenting on your youtube video a month ago about an update you're about to deploy in some time.
Will the midi2music thing also work then?

Thanks!

**Comment by Ocawesome101** [comment]
Hi!

The mscx2music converter is not meant to run in CC---I think it uses newer Lua and it definitely needs at least one luarocks module.  It's also not very good code.

The midi2music utility is a one off experimental thing I wrote to capture midi from a real midi keyboard to the organ format.  It doesn't really work well.

Neither program is intuitive to use since I wrote them for myself and threw them here, not expecting anyone else to want to use them.  (That has clearly proved false, which I'm not mad about at all.  It's nice to see people still taking an interest in the organ.)

I have a rewritten set of software locally that plays MIDI files directly, as well as configures the organ semiautomatically.  I have yet to push it to this repo but I'll probably do that shortly.  Maybe with an updated video too.
